weekly-updates/next.md

```diff
@@ -36,7 +36,10 @@ Solid Events are run by members of the Solid community like you. If you are inte
 |10-10-2019|[Solid Amsterdam](https://www.meetup.com/nl-NL/Solid-Netherlands/events/263745707)|[Jeroen van Beele](https://github.com/jjvbeele)|
 
 ## Solid Implementation
-If you have any new [apps](https://github.com/solid/solid-apps), [Pods](https://github.com/solid/pods), or [identity providers](https://github.com/solid/solid-idp-list) or there are updates to existing Solid implemntations make sure to add them to the list so they get included in next weeks edition. There are no updates on Solid implementations this week. 
+If you have any new [apps](https://github.com/solid/solid-apps), [Pods](https://github.com/solid/pods), or [identity providers](https://github.com/solid/solid-idp-list) or there are updates to existing Solid implemntations make sure to add them to the list so they get included in next weeks edition.
+
+### Solid Data Workers
+NLNet Foundation [has accepted](https://nlnet.nl/project/SOLIDdataworkers/) the application to the NGI0 PET Fund for the Solid Data Workers project, which aims to populate and enrich the personal linked data graph hosted in a Solid pod fetching data from existing sources through dedicated applications (called "Workers"). You can find progresses about the various components involved into the project - still in a very early stage - at https://semantic.builders/
```

Comment:
- `pod fetching` -> `pod by fetching`
- `progresses` -> `progress notes`
- `involved into` -> `involved in`